libs/deck-layers/src/layers/fourwings/FourwingsHeatmapTileLayer.ts

```diff
@@ -123,7 +124,10 @@ export class FourwingsHeatmapTileLayer extends CompositeLayer<
       }
 
       const steps = allValues.map((sublayerValues) =>
```

Comment:
![Screenshot 2024-04-09 at 15 55 05](https://github.com/GlobalFishingWatch/frontend/assets/6906348/b377d8a4-9398-42bc-b45f-05b93b9c99d0)

If any of the sublayers has no data on the current tiles the `ckmeans` calculation crashes the app. Filtering empty sublayers would avoid that situation
```suggestion
      const steps = allValues.filter((sublayer) => sublayer.length).map((sublayerValues) =>
```